aurBuild.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
-# Author: Sjon Hortensius <sjon@hortensius.net>
-# build a package from aur. Extra options will be passed to makepkg
-# usage: aurBuild [--install] [package]
+if [[ "$1" == "--help" ]]; 
```

Comment:
True.
Usually I end the first row of my if statement with ';then', but for the sake of consistency (with your style) I entered the 'then' to the next row but forgot to remove the semicolon.